### PR TITLE
Ignore usage of DOM related typescript type or interface and lint react component that returns fragments

### PR DESCRIPTION
**Pull request by vietanhtran16**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "eslint-plugin-ssr-friendly",
+  "name": "@cultureamp/eslint-plugin-ssr-friendly",
```
Comment (jakubriedl) [inline package.json]: you shouldn't rename package when it is fork. We shouldn't need to publish it to registery and just install from git
```suggestion
  "name": "eslint-plugin-ssr-friendly",
```

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
   "version": "1.0.5",
```
Comment (jakubriedl) [inline package.json]:
```suggestion
  "version": "1.0.6",
```

**Review by jakubriedl — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (vietanhtran16) [reply package.json]: good catch

File: package.json (hunk 1 above)
Comment (vietanhtran16) [reply package.json]: I have put it back now

File: package-lock.json [hunk 3]
```diff
@@ -1,8 +1,2638 @@
 {
```
Comment (jakubriedl) [inline package-lock.json:1]: this package-lock doesn't seem right, it  contain the wrong name and version and also adds ridiculous amount of lines. Isn't that caused by some link or something? can we cleanup the changes here?

Comment (vietanhtran16) [reply package-lock.json:1]: hmm 🤔 that is true. I will check and clean it up

**Comment by vietanhtran16** [comment]
Hey @jakubriedl, I have updated the PR accordingly. I have also added a fix where it should detects errors in component that return fragments (that is why it did not pick up errors previously on VideoPlayer.tsx in kaizen since it returned a fragment)

**Review by jakubriedl — APPROVED** [review]

**Pull request merged**